### ISSUE TITLE
redis-cli up arrow not working for historical commands

### ISSUE THREAD

**Issue by OfirYemini** [microsoftarchive/redis#536]
1. connect to redis via the redis-cli.exe like so: redis-cli -h [serverIp] -p [server port] -c

2. execute some command, for example:  set mykey hello

3.  press the up arrow

expected: command history should be shown
actual: nothing happen, no history is saved

this is a minor but really annoying issue, please fix

**Comment by acexy** [comment]
yes ,I found that the history version will generate a .rediscli_history file under the user directory, which records historical command records, but the new version does not read the file and does not seem to write a similar file

**Comment by OfirYemini** [comment]
any news with this one?
